app/model/command/Commands.scala

```diff
@@ -95,7 +96,7 @@ object Commands {
             Left(CampaignMissingPathPrefix(campaign))
         }
 
-      case None => Left(CampaignNotFound(s"Campaign ID $campaignId not found"))
+      case Left(_) => Left(CampaignNotFound(s"Campaign ID $campaignId not found"))
```

Comment:
This already returns `Left[CampaignNotFound]` though?

Reply:
Good spot - will change.